src/child/components/Favourite.js

```diff
@@ -59,10 +59,11 @@ class Favourite extends Component {
         e.currentTarget.classList.remove('dragOver');
     }
 
-    onDragEnd(e) {
+    onDragEnd(e, stockCode) {
```

Comment:
We can make this look a bit nicer if we use currying:
```
onDragEnd(stockCode) {
  return e => {
    ...
  };
}
```
Then on line 97 we can do:
```
onDragEnd={this.onDragEnd(stockCode)}
```